src/containers/InputContainer.tsx

```diff
@@ -0,0 +1,114 @@
+import React, {useState} from 'react'
+import {v4 as uuidv4} from "uuid";
+import styled from 'styled-components';
+
+import { useAppSelector, useAppDispatch } from '../hooks';
+import { addMessage } from '../state/chatSlice';
+import { MessageType, Client } from '../types';
+
+const Container = styled.div`
+  width: 90vw;
+  max-width: 35rem;
+  height: 4rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 5vh;
+  z-index: 1;
+`
+
+const Form = styled.form`
+  width: 90vw;
+  max-width: 35rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+`
+
+const Button = styled.button`
+  width: 4rem;
+  height: 4rem;
+  background-color: rgba(255, 255, 255, 0.5);
+  border-radius: 4rem;
+  border: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+  &:hover {
+    box-shadow: 0 0 10px rgba(255, 255, 255, 0.6);
+    background-color: rgba(255, 255, 255, 0.8);
+  }
+  & p {
+    font-size: 2.5rem;
+    width: 3rem;
+    text-align: center;
+  }
+`
+
+const Input = styled.input`
+  height: 4rem;
+  width: 70%;
+  border-radius: 4rem;
+  border-color: transparent;
+  padding-left: 1.2rem;
+  outline: none;
+    box-shadow: 0px 0px 20px rgba(255, 255, 255, 1);
+  &:hover {
+    box-shadow: 0 0 10px rgba(255, 255, 255, 0.6);
+  }
+  font-size: 1rem;
+`
+
+const handleTimeFormat = (time: Date):string[] => {
+  const hourString: string = time.getHours().toString(); 
+  const minString: string = time.getMinutes().toString();
+  const timeString: string = (hourString.length === 2? hourString : "0" + hourString) + ": " + (minString.length === 2? minString : "0" + minString)
```

Comment:
```suggestion
  const timeString: string = hourString.padStart(2, "0") + ":" + minString.padStart(2, "0");
```

```padStart```를 사용해서 더 간단하게 표현할 수 있을 것 같아요~!

Reply:
padstart 신기하더라구요, 훨씬 보기 좋은 것 같습니다.